ios/OneginiWrapperErrors.swift

```diff
@@ -30,7 +33,9 @@ enum WrapperError : Error {
         case .malformedUrl:
             return 8010
         case .noProfileAuthenticated:
-            return 8011
+            return 8012
```

Comment:
where that change comes from?

Reply:
This is to match it to the android error code